lab_1/main.py

```diff
@@ -11,7 +11,13 @@ def tokenize(text: str) -> list or None:
     :param text: a text
     :return: a list of lower-cased tokens without punctuation
     """
-    pass
+    if isinstance(text, str):
+        for words in text:
+            if words != ' ' and not words.isalpha():
+                text = text.replace(words, '')
+        tokens = text.lower().split()
+        return tokens
```

Comment:
you can return lower split line right here

Reply:
Done